package.json

```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon ./server.js"
+    "start": "nodemon ./server.js",
+    "sass": "sass ./public/src/css/style.scss ./public/src/css/style.css --style=compressed --embed-sources"
```

Comment:
Would it be possible to write a build script that would perform both sass and typescript compiling, then we could just run `npm run build`?

Reply:
done